ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -120,7 +120,7 @@ test('Verify session created to target with address, then cancel the session', a
   }
 });
 
-test('Verify TCP target is updated', async ({ page }) => {
+test('Verify TCP target is updated @ce @aws @docker', async ({ page }) => {
```

Comment:
I noticed that in some tests which validate features available in both CE and ENT ( e.g. `Log in with invalid password`) we specify both tags. 

Is it intentional that in these target related cases we specify CE tag only?

Reply:
These were the tests that broke in Enterprise because Enterprise has an extra picker to select a TCP or SSH target